library/agent/api/Event.ts

```diff
@@ -123,6 +123,16 @@ type Heartbeat = {
       output: number;
       total: number;
     };
+    routes: {
```

Comment:
we could also add ai usage stats to the Routes object, like we (are planning to) do for the rate limiting stats?